src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -3220,6 +3248,24 @@ private static bool TryScanNumber(string strToConvert, Type resultType, out obje
             return ((Decimal)valueToConvert) != default(Decimal);
         }
 
+        private static bool ConvertBigIntegerToBool(
+            object valueToConvert,
+            Type resultType,
+            bool recursion,
+            PSObject originalValueToConvert,
+            IFormatProvider formatProvider,
+            TypeTable backupTable)
+                => ((BigInteger)valueToConvert) != default(BigInteger);
```

Comment:
Maybe follow the same pattern?
```suggestion
                => ((BigInteger)valueToConvert) != BigInteger.Zero;
```